### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by v-shikhanov**
Was add menu with load, save and close options

### CODE REVIEW

File: src/editor/FileLoader.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+public class FileLoader {
+
+    String loadFile(String fileName)  {
+        File file = new File(fileName);
+
+        /*
+            If file exists and not empty, read it
+         */
+        if(file.isFile()) {
```
Comment (swsms) [inline src/editor/FileLoader.java:17]: a whitepace after **if** is better like **if (**

File: src/editor/TextEditor.java [hunk 2]
```diff
@@ -1,11 +1,115 @@
...
 
 import javax.swing.*;
+import java.awt.*;
 
 public class TextEditor extends JFrame {
+    private FileLoader fileLoader = new FileLoader();
+    static JTextArea textArea = new JTextArea();
+    static JTextField textField = new JTextField();
+
+
```
Comment (swsms) [inline src/editor/TextEditor.java:11]: Why too much spaces?

**Review by swsms — COMMENTED** [review]
In general, it is a good implementation, but there are some style mistakes.

File: src/editor/TextEditor.java (hunk 2 above)
Comment (v-shikhanov) [reply src/editor/TextEditor.java:11]: Yes, I really overuse spaces, I'll pay attention to it in future, thank you!

**Pull request closed without merging**